packages/ckeditor5-widget/theme/widget.css

```diff
@@ -23,6 +23,7 @@
 
 	& .ck-widget__selection-handle {
 		position: absolute;
+		top: 0;
```

Comment:
This should be inside the theme-lark styles, next to those styles: https://github.com/ckeditor/ckeditor5/blob/0b4b2a7b8685521fc5b321dc1dbe1d567f26efb6/packages/ckeditor5-theme-lark/theme/ckeditor5-widget/widget.css#L73-L74

Reply:
It should be moved or copied? Can you provide more details why we need to modify the theme and not the CSS inside the package? I am having hard time finding details on that in the docs.

Reply:
Because some theme might put that handle in some completely different place.